server/config/questions.json

```diff
@@ -46,136 +16,326 @@
-                "hasOther": true,
                 "required": true
             },
             {
-                "name": "grade",
-                "label": "Grade",
-                "type": "radio",
+                "name": "major",
+                "label": "What is your major?",
+                "type": "select",
                 "options": [
-                    "9th grade",
-                    "10th grade",
-                    "11th grade",
-                    "12th grade"
+                    "",
+                    "Accounting",
+                    "Aerospace Engineering",
+                    "Applied Physics",
+                    "Architecture",
+                    "Astronomy or Astrophysics",
+                    "Biochemistry",
+                    "Biology",
+                    "Biomedical Engineering",
+                    "Business",
+                    "Chemical Engineering",
+                    "Chemistry",
+                    "Civil Engineering",
+                    "Computational Media",
+                    "Computer Engineering",
+                    "Computer Science",
+                    "Discrete Mathematics",
+                    "Drama or Theater",
+                    "Earth and Atmospheric Sciences",
+                    "Economics",
+                    "Electrical Engineering",
+                    "English or Literature, Media, and Communication",
+                    "Environmental Engineering",
+                    "Film and Media Studies",
+                    "Finance",
+                    "Global Studies and Languages",
+                    "History",
+                    "Humanities",
+                    "Industrial Design",
+                    "Industrial Engineering",
+                    "International Affairs or International Relations",
+                    "Linguistics",
+                    "Materials Science and Engineering",
+                    "Mathematics",
+                    "Mechanical Engineering",
+                    "Music",
+                    "Music Technology",
+                    "Neuroscience",
+                    "Nuclear and Radiological Engineering",
+                    "Philosophy",
+                    "Physics",
+                    "Political Science",
+                    "Psychology",
+                    "Public Policy",
+                    "Urban Planning"
                 ],
+                "hasOther": true,
                 "required": true
             },
             {
-                "name": "referrer",
-                "label": "How did you hear about us?",
+                "name": "gender",
+                "label": "Gender",
                 "type": "radio",
-                "options": [
-                    "Teacher / Counselor",
-                    "Friend",
-                    "Family"
-                ],
+                "options": ["Male", "Female"],
                 "hasOther": true,
                 "required": true
```

Comment:
Should be optional - either include "Prefer not to answer" or "not require" -  I prefer the later "Prefer not to answer"